### ISSUE TITLE
Class not found when installed with Quarks

### ISSUE THREAD

**Issue by Polkjia** [GameOfLife/WFSCollider-Class-Library#5]
![image](https://user-images.githubusercontent.com/77069555/147478127-5c755ac8-ecc7-4890-8f36-6c35bc2b4537.png)

Windows 10 machine with SC 3.12.1

**Comment by woutersnoei** [comment]
Hmm, this looks more like a generic SC issue. I do see that it says "dirty" in the quark's info window, maybe you deleted the contents of wfs class lib manually but not it's enclosing folder? Perhaps try deleting that too and see what it says then?

**Comment by Polkjia** [comment]
I definitely deleted everything related to WFSC in the folder extensions (~/Local/SuperCollider/Extensions path on Windows). Not sure if there is something somewhere else.

**Comment by woutersnoei** [comment]
On macOS there is a downloadedQuarks folder next to the Extensions folder, the actual quarks are in there. Perhaps on Windows this also exists?

**Comment by Polkjia** [comment]
Yes it does. I deleted manually installed folders and both Unit-Lib and WFSC folder in downloadedQuarks, to then open SC ---> Quarks, install WFSC quark, and still not having classes.

**Comment by Polkjia** [comment]
Ofc I did rebooted interpreter and tried to close and open SC too.

**Comment by woutersnoei** [comment]
hmm, I'm not fully aware of how SC handles these things internally. Perhaps one more thing to try could be deleting the whole downloaded-Quarks folder, recompile and re-download all quarks.

**Comment by Polkjia** [comment]
Tried, it didn't work.

**Comment by woutersnoei** [comment]
Hi, did this pull request solve the issue? https://github.com/GameOfLife/WFSCollider-Class-Library/pull/6
